packages/next-codemod/bin/upgrade.ts

```diff
@@ -263,3 +286,61 @@ async function suggestCodemods(
 
   return codemods
 }
+
+async function suggestReactCodemods(packageManager: PackageManager) {
+  const { runReactCodemod } = await prompts(
+    {
+      type: 'toggle',
+      name: 'runReactCodemod',
+      message: 'Do you want to run the React 19 upgrade codemod?',
+      initial: true,
+      active: 'Yes',
+      inactive: 'No',
+    },
+    { onCancel }
+  )
+
+  if (runReactCodemod) {
+    const commandMap = {
+      yarn: 'yarn dlx',
+      pnpm: 'pnpx',
+      bun: 'bunx',
+      npm: 'npx',
+    }
+    const command = commandMap[packageManager] || 'npx'
+
+    // https://react.dev/blog/2024/04/25/react-19-upgrade-guide#run-all-react-19-codemods
+    execSync(`${command} codemod@latest react/19/migration-recipe`, {
+      stdio: 'inherit',
+    })
+  }
```

Comment:
Let's separate these into input and execution phase like we have before this change. Interleaving prompts and long tasks is bad UX.